addon/helpers/-base.js

```diff
@@ -45,6 +44,12 @@ function helperFactory(formatType, optionalGetValue, optionalReturnEmpty) {
         throw new Error(`format-${formatType} helper requires value`);
       }
 
+      let locale = get(intl, '_locale');
+
+      if (Array.isArray(locale)) {
+        locale = locale[0];
```

Comment:
Use the primary locale always, the only time we use the fallbacks is when resolving translations.